JSONLD/consortia/context.json

```diff
@@ -0,0 +1,8 @@
+{
+    "consortia": {
+        "@context": {
+            "@vocab": "consortia",
+            "@base": "https://github.com/PCMDI/mip-cmor-tables/tree/json-ld/JSONLD/consortia"
```

Comment:
is the json-ld/JSONLD path duplication needed?

Reply:
one is the branch name. This file has been replaced.